code/game/machinery/computer/_computer.dm

```diff
@@ -90,6 +90,16 @@
 		playsound(loc, 'sound/effects/glassbr3.ogg', 100, TRUE)
 		set_light(0)
 
+/obj/machinery/computer/proc/imprint_gps(gps_tag) // Currently used by the upload computers and communications console
+	var/tracker = gps_tag
+	if(!tracker) // Don't give a null GPS signal if there is none
+		return
```

Comment:
This is probably a bad idea but changing it to CRASH should make it more obvious when someone fucked up and forgot to set gps_tag imo
```suggestion
		CRASH("Called imprint_gps without setting gps_tag")
```

Reply:
put [src] at the start of the string

Reply:
same as above